phpBB/includes/functions_acp.php

```diff
@@ -93,16 +93,16 @@ function adm_page_header($page_title)
 
 		'T_ASSETS_VERSION'		=> $config['assets_version'],
 
-		'ICON_MOVE_UP'				=> '<img src="' . htmlspecialchars($phpbb_admin_path) . 'images/icon_up.gif" alt="' . $user->lang['MOVE_UP'] . '" title="' . $user->lang['MOVE_UP'] . '" />',
-		'ICON_MOVE_UP_DISABLED'		=> '<img src="' . htmlspecialchars($phpbb_admin_path) . 'images/icon_up_disabled.gif" alt="' . $user->lang['MOVE_UP'] . '" title="' . $user->lang['MOVE_UP'] . '" />',
-		'ICON_MOVE_DOWN'			=> '<img src="' . htmlspecialchars($phpbb_admin_path) . 'images/icon_down.gif" alt="' . $user->lang['MOVE_DOWN'] . '" title="' . $user->lang['MOVE_DOWN'] . '" />',
-		'ICON_MOVE_DOWN_DISABLED'	=> '<img src="' . htmlspecialchars($phpbb_admin_path) . 'images/icon_down_disabled.gif" alt="' . $user->lang['MOVE_DOWN'] . '" title="' . $user->lang['MOVE_DOWN'] . '" />',
-		'ICON_EDIT'					=> '<img src="' . htmlspecialchars($phpbb_admin_path) . 'images/icon_edit.gif" alt="' . $user->lang['EDIT'] . '" title="' . $user->lang['EDIT'] . '" />',
-		'ICON_EDIT_DISABLED'		=> '<img src="' . htmlspecialchars($phpbb_admin_path) . 'images/icon_edit_disabled.gif" alt="' . $user->lang['EDIT'] . '" title="' . $user->lang['EDIT'] . '" />',
-		'ICON_DELETE'				=> '<img src="' . htmlspecialchars($phpbb_admin_path) . 'images/icon_delete.gif" alt="' . $user->lang['DELETE'] . '" title="' . $user->lang['DELETE'] . '" />',
-		'ICON_DELETE_DISABLED'		=> '<img src="' . htmlspecialchars($phpbb_admin_path) . 'images/icon_delete_disabled.gif" alt="' . $user->lang['DELETE'] . '" title="' . $user->lang['DELETE'] . '" />',
-		'ICON_SYNC'					=> '<img src="' . htmlspecialchars($phpbb_admin_path) . 'images/icon_sync.gif" alt="' . $user->lang['RESYNC'] . '" title="' . $user->lang['RESYNC'] . '" />',
-		'ICON_SYNC_DISABLED'		=> '<img src="' . htmlspecialchars($phpbb_admin_path) . 'images/icon_sync_disabled.gif" alt="' . $user->lang['RESYNC'] . '" title="' . $user->lang['RESYNC'] . '" />',
+		'ICON_MOVE_UP'				=> '<i class="icon acp-icon fa-arrow-circle-up fa-fw" title="' . $user->lang('MOVE_UP') . '" style="color:#4ba5de"></i>',
+		'ICON_MOVE_UP_DISABLED'		=> '<i class="icon acp-icon fa-arrow-circle-up fa-fw" title="' . $user->lang('MOVE_UP') . '" style="color:#d0d0d0"></i>',
+		'ICON_MOVE_DOWN'			=> '<i class="icon acp-icon fa-arrow-circle-down fa-fw" title="' . $user->lang('MOVE_DOWN') . '" style="color:#4ba5de"></i>',
+		'ICON_MOVE_DOWN_DISABLED'	=> '<i class="icon acp-icon fa-arrow-circle-down fa-fw" title="' . $user->lang('MOVE_DOWN') . '" style="color:#d0d0d0"></i>',
+		'ICON_EDIT'					=> '<i class="icon acp-icon fa-cog fa-fw" title="' . $user->lang('EDIT') . '" style="color:#62c046"></i>',
+		'ICON_EDIT_DISABLED'		=> '<i class="icon acp-icon fa-cog fa-fw" title="' . $user->lang('EDIT') . '" style="color:#d0d0d0"></i>',
+		'ICON_DELETE'				=> '<i class="icon acp-icon fa-times-circle fa-fw" title="' . $user->lang('DELETE') . '" style="color:#d74558"></i>',
```

Comment:
The inline style can be removed simply adding the icon colors to the admin CSS

example:

`'ICON_MOVE_UP'				=> '<i class="icon icon-red acp-icon fa-arrow-circle-up fa-fw" title="' . $user->lang('MOVE_UP') . '"></i>',`

then 

```css
.icon-red {
	color: #ff0000;
}
```

![immagine](https://user-images.githubusercontent.com/480857/104140980-a86f0980-53b4-11eb-8dee-dfe54d2fbda9.png)